packages/hs-react-ui/src/components/Modal/Modal.tsx

```diff
@@ -124,7 +124,7 @@ const Modal = ({
           iconPrefix={mdiClose}
           color={closeButtonAttachment === 'inside' ? colors.grayDark : colors.background}
           elevation={closeButtonAttachment === 'inside' ? 0 : 1}
-          type="link"
+          variant="text"
```

Comment:
Do we have constants for these variants?

Reply:
Yeah, good call